collector/transport_node_collector.go

```diff
@@ -81,8 +81,30 @@ func (c *transportNodeCollector) generateTransportNodeStatusMetrics() (transport
 		} else {
 			status = 0
 		}
-		transportNodeStatusMetric := prometheus.MustNewConstMetric(c.transportNodeStatus, prometheus.GaugeValue, status, transportNode.Id, transportNode.DisplayName)
+		transportZoneLabels := c.buildTransportZoneLabels(transportNode)
+		desc := c.buildStatusDesc(transportZoneLabels)
+		transportNodeStatusMetric := prometheus.MustNewConstMetric(desc, prometheus.GaugeValue, status, transportNode.Id, transportNode.DisplayName)
 		transportNodeMetrics = append(transportNodeMetrics, transportNodeStatusMetric)
 	}
 	return
 }
+
+func (c *transportNodeCollector) buildStatusDesc(extraLabels map[string]string) *prometheus.Desc {
```

Comment:
I think we can as well pass `prometheus.Labels` here instead of `map[string]string`